local-cli/generator-macos/templates/macos/Podfile

```diff
@@ -0,0 +1,40 @@
+# require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'
```

Comment:
This is the auto-linking that doesn’t work yet.

Reply:
How are we tracking getting it working?

Reply:
This is up next, I just wanted to make PRs for distinct units of work.